hcx-onboard/src/main/java/org/swasth/hcx/services/OnboardService.java

```diff
@@ -300,6 +302,7 @@ private Map<String, Object> getApplicantBody(OnboardRequest request) {
     }
 
     public ResponseEntity<Object> sendVerificationLink(Map<String, Object> requestBody) throws Exception {
+        Map<String , Object> body =requestBody;
```

Comment:
why you have created one more variable body? can't you get channel from request body itself?